### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hazae41**
pubsub enabled by default

### CODE REVIEW

**Comment by aphelionz** [comment]
Looks good but we probably want to leave a note that says if people are using pre 0.39 then they will still need to enable pubsub.

File: README.md [hunk 1]
```diff
@@ -90,17 +90,18 @@ npm install orbit-db ipfs
...
-// Note that these options need to be passed to IPFS in
-// all examples even if not specified so.
+
+// Create IPFS instance
+
+// For js-ipfs >= 0.38
+const ipfs = new IPFS()
+
+// For js-ipfs < 0.38
 const ipfsOptions = {
```
Comment (RichardLitt) [inline README.md]: I don't think these four lines are needed anymore.

Comment (aphelionz) [reply README.md]: I specifically requested them :D

Comment (RichardLitt) [reply README.md]: Ah. I see the changes. Nevermind! :D

**Pull request merged**